main/ertp/guide/amount-math.md

```diff
@@ -10,111 +10,88 @@ uses `amountMath` methods for all these operations.
 `amountMath` methods also check their arguments' `brand`, 
 throwing an error if the wrong `brand` was used.
 
-An `amountMath` is one of three different kinds, each of which
+An `amountMath` is one of two kinds, each of which
 implements the same methods. Which kind is used for a particular `brand` depends
 on what was specified when the `brand` and its `issuer` were 
 created. The kinds are: 
 - `MathKind.NAT` (`nat`): Used with fungible assets. `amount` `values` are natural numbers (non-negative BigInts).
-- `MathKind.STRING_SET` (`strSet`): Used with non-fungible assets. `amount` `values` are strings.
 - `MathKind.SET` (`set`): Used with non-fungible assets. `amount` `values` are objects or records with multiple properties.
 
-`makeIssuerKit(allegedName, amountMathKind)` creates a new `issuer`,
-`mint`, `brand`, and `amountMath`. 
+`makeIssuerKit(allegedName, amountMathKind, displayInfo=)` creates a new `issuer`,
+`mint`, and `brand`. 
 The second, optional, argument specifies which kind
 of `amountMath` is used for the `brand` in a one-to-one
 association with the new `issuer`. It defaults to `MathKind.NAT`. 
 
+The third, optional, argument specifies how many digits to the right
+of the decimal place to display of values associated with the created
+`brand`. It defaults to `undefined`.
+
```

Comment:
displayInfo includes decimalPlaces, but will include other things as well, so it's not about digits specifically. Let's say it tells the UI how to display the amount.